external-import/doppel/README.md

```diff
@@ -0,0 +1,104 @@
+# OpenCTI Doppel Connector
```

Comment:
add a section for the creation of api key and permission required by the API key in case possible/required.